docs/features/multiple-networks/multi-homing.md

```diff
@@ -47,7 +110,7 @@ spec:
   config: |2
     {
             "cniVersion": "1.0.0",
-            "name": "l3-network",
+            "name": "l3-network-overlay",
```

Comment:
would you tell me why you're updating this name ? 

It's OK, but, this name is as good / bad as the previous one.

Reply:
just following @qinqon 's [request](https://github.com/ovn-org/ovn-kubernetes/pull/4537#pullrequestreview-2207065476) to `modify the example NADs and use different name for the meta.name at NADs and the spec.config.name`

Reply:
yeah, but you can do that using any other name. 

I would go for the typical `tenant<color>` network name ...

Reply:
ACK. DONE